src/m365/spo/commands/listitem/listitem-roleassignment-remove.ts

```diff
@@ -176,12 +139,49 @@ class SpoListItemRoleAssignmentRemoveCommand extends SpoCommand {
       });
 
       if (result.continue) {
-        await removeRoleAssignment();
+        await this.removeRoleAssignment(logger, args);
       }
     }
   }
 
-  private removeRoleAssignment(requestUrl: string, logger: Logger, options: Options): Promise<void> {
+  private async removeRoleAssignment(logger: Logger, args: CommandArgs): Promise<void> {
```

Comment:
Hi @nicodecleyre, in this PR you move quite some local functions to class functions. I like that, it makes it more readable. However, it would be better if we just pass through `args.options` instead of the entire args object. Could you fix that for all instances of this type of refactoring?